### PR TITLE
Move name to orange_livebox, incremental data updates

### PR DESCRIPTION
**Pull request by jacquarg**
* Move name to Orange livebox (Orange request)
* add permission for doctype org.fing.mesinfos.videostream
* ues modified base_konnector, to allow persistence of some usefull values. In that case, lastVideoStreal timestamp, to perform incremental update on each run.
* add env.js file

* ? should we remove permission for doctype files ?

### CODE REVIEW

**Comment by jacquarg** [comment]
Ping @doubleface  ; )

**Review by doubleface — APPROVED** [review]
It looks all good to me. It would be better to find a way to inherit the base konnector instead of replacing it but I don't see it at the moment.

**Comment by doubleface** [comment]
@jacquarg The files permission is still needed since you enhanced base konnector still still checks that the folderPath in cozyFields exists. If your remove that part, the files permission in not needed anymore.

**Comment by jacquarg** [comment]
@doubleface 
I just add a test to make the files permission optional.

I didn't try to enhance base_konnector, as I expected to just upgrade it in konnector-libb with this version ; )

File: base_konnector_with_remember.js [hunk 1]
```diff
@@ -45,6 +45,13 @@ module.exports = {
         .then(account => {
           // get the folder path from the folder id and put it in cozyFields.folderPath
           return new Promise((resolve, reject) => {
+            // quick exit, if no folderPath field.
+            // avoid unecesary file right.
+            console.log(JSON.stringify(cozyFields))
```
Comment (doubleface) [inline base_konnector_with_remember.js]: There is still a console.log that I don't think you want to keep

**Review by doubleface — CHANGES_REQUESTED** [review]

**Review by doubleface — APPROVED** [review]

**Comment by doubleface** [comment]
@jacquarg then it is ok for me

**Comment by doubleface** [comment]
I will merge it when you remove the console.log

**Comment by jacquarg** [comment]
I just did it  : )

**Comment by jacquarg** [comment]
Thanks

**Pull request merged**